### DISCUSSION TITLE
Suggestion to change the license to MIT

### DISCUSSION THREAD

**Discussion by Yankovsky** [Simplify/ibantools#335] [Ideas]
I am not an expert in licensing, but MPL 2.0 seems a bit more restrictive compared to MIT.

**Comment by Simplify** [comment]
I deliberately choose MPL 2.0 for IBANTools as I wanted **copyleft** licence but without placing too many restrictions:

- Using MPL 2.0 code with you code will not force your project into MPL 2.0, like some other copyleft (GPL) licenses do. It covers individual files, not whole project, that is why header in files is important.
- If you use MPL 2.0 code and not distribute it (for example using it in your SaS app server side), you don't need to do anything or worry about MPL 2.0.
- If you use MPL 2.0 in your client side project that is sent to browser, you are distributing software. In that case, you should let users know that part of the software is under MPL 2.0 and point to this repo (or your own repo if you are distributing modified version of the code). That can be done on "about" page for example. Some may call this restriction, but I don't. This is now days very common thing. You can see on every Android phone or TV on about screen that some code is licences under GPL and/or other copyleft licence with a link that provides more info and some source code. If you use some CC licence image on your website, you will depending on licence provide credit to the author. Here is the same thing. Please also note that I have no way to check or enforce this, or willing to do so.

You can also check MPL 2.0 FAQ: https://www.mozilla.org/en-US/MPL/2.0/FAQ/

**Reply by Yankovsky** [reply]
Thank you for the detailed answer!

**Comment by MartinPerrie** [comment]
Hi ... can I just check re: the comment "If you use MPL 2.0 in your client side project that is sent to browser, you are distributing software.". Does that only apply if the code is then accessible outside of your organization? If it is only available inside the organization (e.g. on an intranet or an internal application) then it seems Q6 on the FAQ page applies, even if accessed via a browser. In which case there is no requirement to let users know that part of the software is under MPL 2.0 https://www.mozilla.org/en-US/MPL/2.0/FAQ/.

**Reply by Simplify** [reply]
Yes, that is correct.
